lib/registration.pm

```diff
@@ -135,9 +135,11 @@ sub accept_addons_license {
     # license (see bsc#1089163) and so are not be shown twice
     push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless (is_sle('15+') || is_sle_micro);
     # HA and WE have licenses when calling yast2 scc
-    push @addons_with_license, @SLE15_ADDONS_WITH_LICENSE_NOINSTALL if (is_sle('15+') and is_sle('<=15-sp4') and get_var('IN_PATCH_SLE'));
+    push @addons_with_license, @SLE15_ADDONS_WITH_LICENSE_NOINSTALL if (is_sle('15+') and get_var('IN_PATCH_SLE'));
     # HA does not show EULA when doing migration to 12-SP5
     @addons_with_license = grep { $_ ne 'ha' } @addons_with_license if (is_sle('12-sp5+') && get_var('UPGRADE') && !get_var('IN_PATCH_SLE'));
+    # HA does not show EULA when doing migration from 15-SP5 to 15-SP6
+    @addons_with_license = grep { $_ ne 'ha' } @addons_with_license if (is_sle('=15-sp5') && get_var('UPGRADE'));
```

Comment:
I  think it is better to set it as '15-sp5+' to cover SP5, SP6, etc. and with the condition of "&& !get_var('IN_PATCH_SLE'))" .  'IN_PATCH_SLE' used to judge the stage of migration process.

Reply:
I cannot add the !get_var('IN_PATCH_SLE') because we need it to be 1 for the we eula. Plus, if I change the IN_PATCH_SLE to 0 for 15_sp5 I will have to add this kind of confusing conditions above in the code as well. I changed it to 15-sp5+, which for now doesn't really offer anything because we do not do migrations from 15-sp6, but in the future, if the eula conditions don't change, it will be useful.

Reply:
I can understand this, just don't know whether this will cause failure if it is not eula.